pkgs/by-name/fa/factorio_achievement_enabler/package.nix

```
@@ -0,0 +1,34 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  lib,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "fae_linux";
+  version = "1.3";
+
+  src = fetchFromGitHub {
+    owner = "UnlegitSenpaii";
+    repo = "FAE_Linux";
+    rev = "v${version}";
+    hash = "sha256-lm/s9rc4/2TIT2mzIPwdFoPB9GZm4qluK2yVoL7KwnE";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  installPhase = ''
+    runHook preInstall
+    mv out $out
+    runHook postInstall
+  '';
+
+  meta = {
+    description = "Factorio Achievement Enabler for Linux, with this you can get steam achievements while playing the game with mods";
```

Descriptions should be simple and concise
```suggestion
    description = "Factorio achievement enabler for Linux";
```